compiler/src/dmd/declaration.h

```diff
@@ -720,7 +720,6 @@ class FuncDeclaration : public Declaration
     bool isAbstract() override final;
     bool isSafe();
     bool isTrusted();
-    bool isNogc();
```

Comment:
this is unused in both LDC and GDC (not to be confused with `TypeFunction.isNogc` which is used in GDC)